### DISCUSSION TITLE
Error run phyton file in terminal

### DISCUSSION THREAD

**Discussion by centeiol** [microsoft/vscode-discussions#708] [Extension Development QnA]
Hello !!

When running the python script in VScode terminal I get an error
I have already uninstalled and installed VScode and phyton without success.
Error below.

![image](https://github.com/microsoft/vscode-discussions/assets/25835068/a5c1ff92-6b3f-4054-bd5b-906f2fb3fdc8)

I have also put the path in the windows ENV , without success
and I put the following settings , without success

![image](https://github.com/microsoft/vscode-discussions/assets/25835068/2248f763-bfa7-4d09-be87-a42b51ec9a94)

How to solve it?
Regards

**Comment by gjsjohnmurray** [comment]
This forum is intended for people who are developing VS Code extensions. Please seek Python coding assistance elsewhere.
